### DISCUSSION TITLE
deepin-system-monitor-plugin-popup.desktop字段名NotShowIn拼写错误

### DISCUSSION THREAD

**Discussion by hotime** [linuxdeepin/developer-center#3250] [Q&A | 问答板块] [answered]
deepin-system-monitor-5.9.17中的文件deepin-system-monitor-plugin-popup.desktop第4行字段名NotShowIn错误的拼写为了NoShowIn，请注意修正。
涉及到该错误的包有：
https://community-packages.deepin.com/deepin/pool/main/d/deepin-system-monitor/deepin-system-monitor_5.9.17-1_amd64.deb
https://community-packages.deepin.com/deepin/pool/main/d/deepin-system-monitor/deepin-system-monitor_5.9.17-1_i386.deb
https://community-packages.deepin.com/deepin/pool/main/d/deepin-system-monitor/deepin-system-monitor_5.9.17.orig.tar.xz

**Comment by BLumia** [comment, marked as answer]
这类问题其实可以开 issue 的。另外我们目前接受来自社区的代码贡献了，也可以考虑发起 PR 进行修改 :)

**Reply by kt286** [reply]
https://github.com/linuxdeepin/deepin-system-monitor/pull/120
